dev/com.ibm.ws.logstash.collector/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -3,8 +3,7 @@
 #NLS_MESSAGEFORMAT_NONE
 
 logstashCollector=Logstash Collector
-logstashCollector.desc=Logstash collector gathers data from various sources and forwards the data to a logstash server using Lumberjack protocol.
-
+logstashCollector.desc=Logstash collector internally collects log data at run time and sends it directly to a Logstash server using Lumberjack protocol.
```

Comment:
on second thought... might be better to say...

Logstash collector internally collects log and other event data at run time and sends it directly to a Logstash server using Lumberjack protocol.